romancal/regtest/test_resample.py

```diff
@@ -106,9 +107,45 @@ def test_resample_single_file(rtdata, ignore_asdf_paths):
         for x in ["data", "err", "var_poisson", "var_rnoise", "var_flat"]
     )
 
+    step.log.info(
+        f"""DMS343 MSG: Were the variance arrays populated (variance propagation)? :\
+            {
+                all(
+                    np.sum(~np.isnan(getattr(resample_out, x))) for x in [
+                        "var_poisson",
+                        "var_rnoise",
+                    ]
+                )
+            }"""  # noqa: E501
+    )
+    assert all(
+        np.sum(~np.isnan(getattr(resample_out, x)))
+        for x in ["var_poisson", "var_rnoise"]
+    )
+
+    step.log.info(
+        f"""DMS343 MSG: Are there NaNs or zeros in the data and variance arrays? :\
```

Comment:
It looks like this is only checking the variance arrays, which I think is appropriate.  Let's delete 'data and' and add some verbiage so that it more obviously connects to the requirement text.
```suggestion
        f"""DMS343 MSG: Are there NaNs or zeros in the variance arrays, indicating poor data quality? :\
```